File: app/config/time.py
from datetime import datetime

def get_greeting():
    hour = datetime.now().hour

    if 5 <= hour < 11:
        return "Selamat Pagi 🌅"
    elif 11 <= hour < 15:
        return "Selamat Siang ☀️"
    elif 15 <= hour < 18:
        return "Selamat Sore 🌇"
    else:
        return "Selamat Malam 🌙"

File: src/config/time.js
export function getGreeting() {
  const hour = new Date().getHours();
  if (hour >= 5 && hour < 11)  return 'Selamat Pagi 🌅';
  if (hour >= 11 && hour < 15) return 'Selamat Siang ☀️';
  if (hour >= 15 && hour < 18) return 'Selamat Sore 🌇';
  return 'Selamat Malam 🌙';
}
